<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerTextActionsPopup</class>
 <widget class="QWidget" name="ViewerTextActionsPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="arrowCont" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>19</height>
      </size>
     </property>
     <widget class="QLabel" name="topArrowLbl">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>2</y>
        <width>24</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="popupCont">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000</width>
       <height>10000</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="actionsCont" native="true">
        <layout class="QHBoxLayout" name="actions22Cont">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="createNoteBtn">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Crear Nota</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="highlightedBtn">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Resaltar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchBtn">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="bqQTextBrowser" name="definitionTextBrowser">
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="seeCompleteDefinition">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Diccionarios</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="wikipediaPopupBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="arrowBottomCont" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QLabel" name="bottomArrowLbl">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>24</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>bqQTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header location="global">bqQTextBrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
